modules/bolt/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "puppetlabs/bolt",
+    "version": "0.7.0",
```

Comment:
We probably need something that bumps this or it should be left out.

Reply:
I used what we use as minimum meta data in unit tests in puppet. Don't know what we can exclude, or if we need the metadata.json at all.

Reply:
I think we can live without maybe here since it's going to be vendored